framework/grid/DataColumn.php

```diff
@@ -189,6 +189,12 @@ protected function renderFilterCellContent()
             if (is_array($this->filter)) {
                 $options = array_merge(['prompt' => ''], $this->filterInputOptions);
                 return Html::activeDropDownList($model, $this->attribute, $this->filter, $options) . $error;
+            } elseif ($this->format === 'boolean') {
+                $options = array_merge(['prompt' => ''], $this->filterInputOptions);
+                return Html::activeDropDownList($model, $this->attribute, [
+                        $this->grid->formatter->booleanFormat[0],
+                        $this->grid->formatter->booleanFormat[1]
+                    ], $options) . $error;
```

Comment:
I thought it's not in internal docs code style but it is there - I'll fix it as well, cheers.